x11-wm/2bwm/2bwm-0.2.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3 savedconfig
```

Comment:
This is still a live ebuild. Drop git-r3 and take the tar.gz snapshot off https://github.com/venam/2bwm/releases